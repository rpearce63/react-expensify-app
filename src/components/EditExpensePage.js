import React from 'react';

const EditExpensePage = (props) => {
    console.log(props);
   return (
        <div>
            Edit expense with id {props.match.params.id}
        </div>
    )
};

export default EditExpensePage;