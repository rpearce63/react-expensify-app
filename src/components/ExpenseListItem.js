import React from 'react';

const ExpenseListItem = ({description, amount, createdAt}) => (
    <li>{description} - {amount} - {createdAt}</li>
);

export default ExpenseListItem;