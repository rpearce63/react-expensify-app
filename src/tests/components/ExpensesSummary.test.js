import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';
import expenses from '../fixtures/expenses';

test('should render page for a single expense correctly',() => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235}/>);
    expect(wrapper).toMatchSnapshot();
}); 

test('should render page for multiple expenses correctly',() => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={235123}/>);
    expect(wrapper).toMatchSnapshot();
}); 