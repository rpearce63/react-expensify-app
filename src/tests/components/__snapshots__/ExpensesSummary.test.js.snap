// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render page for a single expense correctly 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`should render page for multiple expenses correctly 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $2,351.23
  </h1>
</div>
`;
